app/boards/arm/glove80/glove80_lh_defconfig

```diff
@@ -16,16 +16,43 @@ CONFIG_USB_DEVICE_VID=0x16c0
 CONFIG_USB_DEVICE_MANUFACTURER="MoErgo"
 CONFIG_USB_DEVICE_SN="moergo.com:GLV80-0123456789ABCDEF"
 
+CONFIG_BT_DEVICE_NAME="Glove80"
+
 CONFIG_BT_DIS_PNP_PID=0x27db
 CONFIG_BT_DIS_PNP_VID=0x16c0
 CONFIG_BT_DIS_MANUF="MoErgo"
 CONFIG_BT_DIS_MODEL="Glove80"
 
+### Bluetooth configuration workarounds
+
+# Use higher radio transmit power
 CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
 
+# Use non-legacy LLCP (required for compatibility with recent Linux kernel)
```

Comment:
I'm not sure this will work anymore because of the following from `app/Kconfig`:

```
choice BT_LL_SW_LLCP_IMPL
    default BT_LL_SW_LLCP_LEGACY if !ZMK_BLE_EXPERIMENTAL_CONN
```

We need to use `ZMK_BLE_EXPERIMENTAL_CONN` rather than set all the flags individually so `BT_LL_SW_LLCP_IMPL` will remain the default (`CONFIG_BT_LL_SW_LLCP`).

Reply:
As discussed on Discord, my understanding of this was wrong. You can safely ignore this comment.